examples/platforms/da15000/radio.c

```diff
@@ -95,15 +95,17 @@ void da15000RadioInit(void)
 void otPlatRadioGetIeeeEui64(otInstance *aInstance, uint8_t *aIeeeEui64)
 {
     (void)aInstance;
+    uint32_t *timeStamp  = (uint32_t *) 0x7F8EA08;
+    uint32_t *testId     = timeStamp + 1;
```

Comment:
Can you make this an explicit address, preferably using a `#define`? Just that would make it clearer what is happening here.